content/en/docs/tasks/security/authorization/authz-td-migration/index.md

```diff
@@ -112,7 +112,7 @@ Notice that it may take tens of seconds for the authorization policy to be propa
     Istio mesh is now running with a new trust domain, `new-td`.
```

Comment:
Should we state the assumption of the prior settings. In the other words, set controlPlaneSecurityEnabled and mtls.enabled is not part of the change (though they should be on to make sense). 

We also don't want to mentioned both of these flags, as likely they will be removed in 1.6. User should set authn policy for mtls.enabled instead.

Reply:
As this PR focuses on Citadel related changes, this change on the flags of controlPlaneSecurityEnabled and mtls.enabled can be in a separate PR.

Reply:
I think we should remove in the same CL to avoid create any side effect. The whole command can be simplify to a oneliner with --ser

Reply:
A simplification sounds good to me. Can you suggest the oneliner to use?

Reply:
I think this should work

`istioctl manifest apply --set values.global.trustDomain=new-td`

You can mention this assumes user previously install Istio with default profile.

Reply:
Simplified to "istioctl manifest apply --set profile=demo --set values.global.trustDomain=new-td". The profile used in the guide is demo, hence the --set profile=demo.

Reply:
It's ok to use profile `demo` if you want, but you should also change step 2  to the similar command.

Reply:
And also in the section below (with alias)

Reply:
The step 2 has been changed to use a similar command. As to the following trustDomainAliases, I am not sure about the equivalent command line since it has an array in it. In addition, td-installation.yaml is also referred in the cleanup section.  
    apiVersion: install.istio.io/v1alpha1
    kind: IstioOperator
    spec:
      values:
        global:
          controlPlaneSecurityEnabled: false
          mtls:
            enabled: true
          trustDomain: new-td
          trustDomainAliases:
            - old-td

Reply:
Hmm. I don't know how to set array params neither. Keep the yaml is fine for now, but please remove mtls and controlPlaneSecurity to be consistent with other.

Reply:
mtls and controlPlaneSecurity have been removed.